package on2020_05.on2020_05_14_Codeforces___Codeforces_Round__429__Div__1_.E__In_a_Trap1;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("src/test/java/on2020_05/on2020_05_14_Codeforces___Codeforces_Round__429__Div__1_/E__In_a_Trap1/E. In a Trap.json"))
			Assert.fail();
	}
}
